include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -9,15 +9,33 @@
 
 namespace onnxruntime {
 
+/**
+* Default_Global refers to all the transformations which are required for 
+  corrcetness and are execution provider independent
+* Default_ProvderSpecific refers to transformations which are also required 
+  for correctness but are execution provider specific
+* Optional_L1 refers to all basic optmizations. These are run both before and 
+  after graph partitioning
```

Comment:
Should this be more generic and say 'basic transformations' given this is a GraphTransformer not a GraphOptimizer?

Reply:
Agree